compiler.md

```diff
@@ -107,6 +107,28 @@ import MyTag from './path/to/tags/my-tag.riot'
 component(MyTag)(document.getElementById('root'))
 ```
 
+### Typescript support
+
+The Riot.js compiler supports typescript syntax out of the box. 
+If you want to enable the type checking for your component API and expressions you will export its `RiotComponent` interface as it follows:
+
+```html
+<my-component>
+  <!-- notice that lang="ts" is optional and eventually needed just for your IDE code highlighting -->
+  <script lang="ts">
+    import { RiotComponent } from 'riot'
+    
+    export interface MyComonent extends RiotComponent<MyComponentProps, MyComponentState> {
```

Comment:
Typo error here

Reply:
could you please PR?

Reply:
https://github.com/riot/riot.github.io/pull/309